### DISCUSSION TITLE
Can you change the fill color for each value in a column series?

### DISCUSSION THREAD

**Discussion by upswing1** [Live-Charts/LiveCharts2#1186] [Q&A]
Can someone help me how to figure out this problem.
In order to display the labels on the x-axis I have created a column series containing some values. Each column in the bar chart is displayed with the same fill color. I need to customize the fill color and the stroke color for each value on the series. Can it be done?

As a second test I have created a series containing multiple column series. Each column series contains a single value. Using this method I can change the fill color and stroke color for each column series. Since the column contains a single element the only label displayed is the first one in a label array. It makes sense but I need to to display a label under each column series. Is there a way to override what gets displayed on the x axis?  

Thanks

**Comment by beto-rodriguez** [comment]
This is now easy on the latest version:

https://livecharts.dev/docs/WPF/2.0.0-beta.950/samples.general.mapPoints

**Comment by upswing1** [comment]
Thank you for your answer,  I am running now the latest version and I get the colors to show in the fill but not in the Stroke,

Is the current syntax correct?:
.OnPointMeasured(point =>
{
    // this method is called for each point in the series
    // we can customize the visual here

    if (point.Visual is null)
    {
        return;
    }

    // get a paint from the array
    var paint = FillColors[point.Index % FillColors.Length];
    // set the paint to the visual
    point.Visual.Fill = paint;

    var paint2 = StrokeColors[point.Index % FillColors.Length];
    // set the paint to the visual
    point.Visual.Stroke = paint2;
});

I have also tried:

               var series = new ColumnSeries<double>
               {
                   Values = values, // new[] { 2, 5, 4, 6, 8, 3, 2, 4, 6 },
                   DataLabelsPaint = new SolidColorPaint(new SKColor(30, 30, 30)),
                   DataLabelsPosition = DataLabelsPosition.Top
               }

               .OnPointMeasured(point =>
               {
                   // this method is called for each point in the series
                   // we can customize the visual here

                   if (point.Visual is null)
                   {
                       return;
                   }

                   // get a paint from the array
                   var paint = FillColors[point.Index % FillColors.Length];
                   // set the paint to the visual
                   point.Visual.Fill = paint;

                   
               })

               .OnPointMeasured(point =>
               {
                   // this method is called for each point in the series
                   // we can customize the visual here

                   if (point.Visual is null)
                   {
                       return;
                   }

                   
                   var paint2 = StrokeColors[point.Index % FillColors.Length];
                   // set the paint to the visual
                   point.Visual.Stroke = paint2;
               });
